ci/infra/azure/registration.auto.tfvars

```diff
@@ -3,10 +3,11 @@
 # - register against local SUSE Repository Mirroring Server
 
 # SUSE CaaSP Product Registration Code
-# caasp_registry_code = ""
+#caasp_registry_code = ""
+caasp_registry_code = "INTERNAL-USE-ONLY-8134-cc01"
```

Comment:
This should not be leaked :)

Please reach out to the SCC team to have this code removed.